rectype,issueid,project_owner,project_name,actor,time,text,action,title,clean_text,Test
rectype,issueid,project_owner,project_name,actor,time,text,action,title,text,N
issue_title,408,pycqa,astroid,rogalski,2017-02-28 21:19:38,Fixes PyCQA/pylint#1348,start issue,builder: fix detection of whether found file is a package when there is a __init__.py in file name,fix pycqapylint1348,N
issue_closed,408,pycqa,astroid,rogalski,2017-04-02 11:55:28,nan,closed issue,builder: fix detection of whether found file is a package when there is a __init__.py in file name,nan,N
pull_request_title,408,pycqa,astroid,rogalski,2017-02-28 21:19:38,Fixes PyCQA/pylint#1348,093f16b1f35464f84203689d94710cb72367f9eb,builder: fix detection of whether found file is a package when there is a __init__.py in file name,fix pycqapylint1348,N
pull_request_merged,408,pycqa,astroid,rogalski,2017-04-02 11:55:28,builder: fix detection of whether found file is a package when there is a __init__.py in file name,92665fe273b9e5899fa28ef7f67d4794db42478f,Pull request merge from rogalski/astroid:builder_fix_detection_of_package to PyCQA/astroid:master,builder fix detect whether found file packag initpi file name,N
issue_comment,408,pycqa,astroid,PCManticore,2017-03-01 09:23:36,"I don't think that this is the root cause of that bug. In the reproducing example provided there, the package had a ``__init__.py`` file. We should check where astroid is raising the ImportError in the first place, in order to know what triggers the bug.",nan,nan,I dont think root caus bug In reproduc exampl provid packag initpi file We check astroid rais importerror first place order know trigger bug,N
issue_comment,408,pycqa,astroid,rogalski,2017-03-05 09:20:08,"@PCManticore Well, all I know is that it fixes reported issue. I'm just no sure where is a place for an automated test.",nan,nan,pcmanticor well I know fix report issu Im sure place autom test,Y
pull_request_commit,408,pycqa,astroid,rogalski,2017-02-28 21:19:22,"builder: fix detection of whether found file is a package when there is a __init__.py in file name

Fixes PyCQA/pylint#1348",d8791ff449ef5cb8518a9ac0e59c1084842fdf7d,nan,builder fix detect whether found file packag initpi file name fix pycqapylint1348,N
pull_request_commit,408,pycqa,astroid,rogalski,2017-03-27 19:56:47,Add unit test,5a88003f5c366c81cf209cb7d8faa8dcb76efde7,nan,add unit test,N
pull_request_commit,408,pycqa,astroid,rogalski,2017-03-27 19:57:06,Merge branch 'master' into builder_fix_detection_of_package,f1257b9265eea33a0969ca45f790dd2e6090653c,nan,merg branch master builderfixdetectionofpackag,N
pull_request_commit,408,pycqa,astroid,rogalski,2017-04-02 11:42:47,remove not needed test,f0dd1d23dddfe5ed571a73d6e96d2e9e5805ca77,nan,remov need test,N
pull_request_commit,408,pycqa,astroid,rogalski,2017-04-02 11:48:21,remove not needed empty line removal,093f16b1f35464f84203689d94710cb72367f9eb,nan,remov need empti line remov,N
